pkg/systemd/overview.less

```diff
@@ -316,3 +316,17 @@
 .pf-c-button.no-left-padding {
     padding-left: 0;
 }
+
+.pf-c-alert.no-title h4 {
+    font-weight: normal;
+    color: black;
+    font-family: inherit;
+}
```

Comment:
We don't seem to override that on other places that use Alert. Why do we need that? @marusak , did you add that?

Reply:
I am stretching this alert a bit - meaning that in one case I want the 'title' to look like a normal text.